tests/testCase1.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "TestCase",
+  "uuid": "4fa03a8d-2e39-4866-9cd3-69b77bd78a6b",
+  "testName": "t test produces calibrated p values",
+  "keywords": ["t-test", "p value", "statistics"],
+  "description": "This test checks that the p values produced by stats::t.test do not deviate substantially from the expected uniform distribution.",
```

Comment:
should render to markdown

Reply:
GAMP and CSA suggest a Test Objective, which ist linked to N Test cases.